docs/linter_settings.rst

```diff
@@ -30,16 +30,17 @@ The value may be a string or an array. If it is a string, it will be parsed as i
 The default value is an empty array.
 
 
-chdir
+working_dir
 -----
 This setting specifies the linter working directory.
 
-The value must be a string, corresponding to a valid directory path.
+The value must be a string, corresponding to a valid directory path. Variable
+substitutions are supported.
```

Comment:
Does this need an explicit mention? We replace substitutions in all settings.

Reply:
Done